core/src/main/java/com/opensymphony/xwork2/interceptor/ChainingInterceptor.java

```diff
@@ -160,7 +160,20 @@ private void copyStack(ActionInvocation invocation, CompoundRoot root) {
         Map<String, Object> ctxMap = invocation.getInvocationContext().getContextMap();
         for (Object object : list) {
             if (shouldCopy(object)) {
-                reflectionProvider.copy(object, invocation.getAction(), ctxMap, prepareExcludes(), includes);
+                String actionConfigClassName = invocation.getProxy().getConfig().getClassName();
```

Comment:
> editable - the class (or interface) to restrict property setting to i.e. only setting properties defined in the given "editable" class (or interface))
I learnt this name from Spring docs. User also can set a class or interface which is not source class nor target class but an encapsulation of desired properties inside it.